<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualSetDialog</class>
 <widget class="QDialog" name="ManualSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>610</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Manual Set</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>150</y>
     <width>575</width>
     <height>305</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>575</width>
     <height>305</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>610</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>610</width>
     <height>42</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="qCupTypelabel_2">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>10</y>
      <width>100</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>Cup Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CupTypecomboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>120</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>24</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="qItemCombinationlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>24</height>
     </size>
    </property>
    <property name="text">
     <string>Item Combination</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ItemCombinationcomboBox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>24</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>477</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>477</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="qCupTypelabel">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>112</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cup Type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="qCupTypeNameLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>112</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="qProjectProtfoliolabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>112</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Protfolio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="qProjectCountlabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>112</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="qProjectCountNamelabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>112</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="qProjectProtfolioNameLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>112</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
